pkgs/os-specific/darwin/apple-source-releases/adv_cmds/xcode.nix

```diff
@@ -0,0 +1,21 @@
+{stdenv, fetchurl, xcbuild}:
+
+stdenv.mkDerivation {
```

Comment:
Either way, very exciting to see the direction it's going 😄 thanks for doing this!